src/chrome/chromeDebugAdapter.ts

```diff
@@ -1332,6 +1337,7 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         }
     */
     public disconnect(args: DebugProtocol.DisconnectArguments): void {
+        telemetry.reportEvent('FullSessionStatistics/SourceMaps/Overrides', { aspNetClientAppFallbackCount: sourceMapUtils.getAspNetFallbackCount() });
```

Comment:
I don't recall it but did Michael find issues with telemetry reporting after we issue the disconnect request? Can you confirm we will get this at all instances?

Reply:
I tested this a few times on my machine and the telemetry did appear on the telemetry test tool.

Reply:
Cool thanks!